File: spring-data-requery/src/main/java/org/springframework/data/requery/repository/support/RequeryRepositoryImplementation.java
package org.springframework.data.requery.repository.support;

import org.springframework.data.requery.repository.RequeryRepository;

/**
 * com.coupang.springframework.data.requery.repository.support.RequeryRepositoryImplementation
 *
 * @author debop
 * @since 18. 6. 6
 */
public interface RequeryRepositoryImplementation<T, ID> extends RequeryRepository<T, ID> {

    void setRepositoryMethodMetadata(CrudMethodMetadata crudMethodMetadata);

}

File: spring-data-requery/src/main/kotlin/org/springframework/data/requery/repository/support/RequeryRepositoryImplementation.kt
package org.springframework.data.requery.repository.support

import org.springframework.data.requery.repository.RequeryRepository

/**
 * RequeryRepositoryImplementation
 *
 * @author debop@coupang.com
 */
interface RequeryRepositoryImplementation<E: Any, ID>: RequeryRepository<E, ID> {

    fun setRepositoryMethodMetadata(crudMethodMetadata: CrudMethodMetadata?)
}
